src/quill.imageCompressor.ts

```diff
@@ -68,8 +68,8 @@ class imageCompressor {
     });
   }
 
-  async fileChanged() {
-    const files = this.fileHolder?.files;
+  async fileChanged(files?: File[]) {
```

Comment:
@Ckpuna4 why is this property optional? Are you using this externally?

Reply:
>why is this property optional?

To preserve backwards compatibility, if someone is already using this method in their project

>Are you using this externally?

After upgrading to the latest version — yes